### DISCUSSION TITLE
N3DS- Any way to split stacks?

### DISCUSSION THREAD

**Discussion by DrakeVanLyon** [diasurgical/DevilutionX#3725] [Q&A]
I have been enjoying Diablo+Hellfire on the N3DS for awhile now, however, there is no on-screen keyboard or anything from my experience. I was trying to split a stack to see if it was possible, but whenever it asks me to input a number I have nothing that I can do for input. Is there a way around this? Or are there any plans to add an on-screen keyboard to the bottom screen or something?

Unrelated question- Any tips on beating Diablo (Boss) with a Sorcerer? I'm struggling like Hell right now lol.

**Comment by StephenCWills** [comment]
I'd say that #2249 is the closest thing we have to a plan for splitting stacks on 3DS. We could potentially use the keyboard applet like we do in the menus, but it locks up the main thread. That could cause some problems, particularly for multiplayer games.

For Diablo, you should try Holy Bolt.
